mmv1/products/dialogflowcx/Environment.yaml

```diff
@@ -55,12 +54,20 @@ custom_code: !ruby/object:Provider::Terraform::CustomCode
   pre_update: templates/terraform/pre_create/dialogflow_set_location.go.erb
   pre_delete: templates/terraform/pre_create/dialogflow_set_location.go.erb
   pre_read: templates/terraform/pre_create/dialogflow_set_location.go.erb
+custom_diff:
+  - 'tpgresource.DefaultProviderProject'
 examples:
   - !ruby/object:Provider::Terraform::Examples
     name: 'dialogflowcx_environment_full'
     primary_resource_id: 'development'
     vars:
       agent_name: 'dialogflowcx-agent'
+  - !ruby/object:Provider::Terraform::Examples
+    skip_docs: true
+    name: 'dialogflowcx_environment_regional'
+    primary_resource_id: 'development'
+    vars:
+      agent_name: 'issue12880'
```

Comment:
Could you add a `-` in this name (ie. `issue-12880`) so that our tooling can clean up these resources?

Reply:
Copied existing test 😅. Done!